app/src/main/kotlin/com/twain/interprep/data/model/Interview.kt

```diff
@@ -98,7 +99,7 @@ sealed class DashboardInterviewType(
         DashboardInterviewType(BackgroundLightGray, BackgroundDarkGray, CARD_FULL_WIDTH_FACTOR)
 }
 
-fun Interview.isValid() = listOf(date, time, company).none { it.isEmpty() }
+fun Interview.isValid() = listOf(date, time, company).none { it.isEmpty() } and listOf(jobPostLink, companyLink).all { it.isBlank() or isValidURL(it) }
```

Comment:
```suggestion
fun Interview.isValid(): Boolean {
    val requiredInfoValid = listOf(date, time, company).all { it.isNotEmpty() }
    val linksValid = listOf(jobPostLink, companyLink).all { it.isBlank() or isValidURL(it) }
    return requiredInfoValid and linksValid
}
```

For better code readability ?